examples/vpc_peering_common_dual_stack/variables.tf

```diff
@@ -0,0 +1,369 @@
+
+  Multiple keys can be added and will be deployed by the code.
+
+  EOF
+  type        = map(any)
+  default     = {}
+}
+
+#VPC
+
+variable "networks" {
+  description = <<-EOF
+  A map containing each network setting.
+
+  Example of variable deployment :
+
+  ```
+  networks = {
+    fw-mgmt-vpc = {
+      vpc_name = "fw-mgmt-vpc"
+      create_network = true
+      delete_default_routes_on_create = false
+      mtu = "1460"
+      routing_mode = "REGIONAL"
+      subnetworks = {
+        fw-mgmt-sub = {
+          name = "fw-mgmt-sub"
+          create_subnetwork = true
+          ip_cidr_range = "10.10.10.0/28"
+          region = "us-east1"
+        }
+      }
+      firewall_rules = {
+        allow-mgmt-ingress = {
+          name = "allow-mgmt-vpc"
+          source_ranges = ["10.10.10.0/24", "1.1.1.1/32"] # Replace 1.1.1.1/32 with your own souurce IP address for management purposes.
+          priority = "1000"
+          allowed_protocol = "all"
+          allowed_ports = []
+        }
+      }
+    }
+  }
+  ```
+
+  For a full list of available configuration items - please refer to [module documentation](https://github.com/PaloAltoNetworks/terraform-google-swfw-modules/tree/main/modules/vpc#input_networks)
+
+  Multiple keys can be added and will be deployed by the code.
+  EOF
+}
+
+variable "vpc_peerings" {
+  description = <<-EOF
+  A map containing each VPC peering setting.
+
+  Example of variable deployment :
+
+  ```
+  vpc_peerings = {
+    "trust-to-spoke1" = {
+      local_network_key = "fw-trust-vpc"
+      peer_network_key  = "fw-spoke1-vpc"
+
+      local_export_custom_routes                = true
+      local_import_custom_routes                = true
+      local_export_subnet_routes_with_public_ip = true
+      local_import_subnet_routes_with_public_ip = true
+
+      peer_export_custom_routes                = true
+      peer_import_custom_routes                = true
+      peer_export_subnet_routes_with_public_ip = true
+      peer_import_subnet_routes_with_public_ip = true
+    }
+  }
+  ```
+  For a full list of available configuration items - please refer to [module documentation](https://github.com/PaloAltoNetworks/terraform-google-swfw-modules/tree/main/modules/vpc-peering#inputs)
+
+  Multiple keys can be added and will be deployed by the code.
+  EOF
+  type        = map(any)
+  default     = {}
+}
+
+variable "routes" {
+  description = <<-EOF
+  A map containing each route setting. Note that you can only add routes using a next-hop type of internal load-balance rule.
+
+  Example of variable deployment :
+
+  ```
+  routes = {
+    "default-route-trust" = {
+      name = "fw-default-trust"
+      destination_range = "0.0.0.0/0"
+      vpc_network_key = "fw-trust-vpc"
+      lb_internal_name = "internal-lb"
+    }
+  }
+  ```
+
+  Multiple keys can be added and will be deployed by the code.
+  EOF
+  type        = map(any)
+  default     = {}
+}
+
+variable "policy_routes" {
+
```

Comment:
Please add a description, type and default. Same for the next var.

Reply:
Added.